backend/uclapi/roombookings/helpers.py

```diff
@@ -308,8 +337,8 @@ def _filter_for_free_rooms(all_rooms, bookings, start, end):
     for room in rooms_with_bookings:
         roomid, siteid = room["roomid"], room["siteid"]
         if (
```

Comment:
Here I would use the `.get(key, [default])` variant too

Reply:
I'm not sure I wrote this code but we might as well fix it up now.